server/src/modules/rules/helpers/rule.comparator.service.ts

```diff
@@ -436,11 +438,11 @@ export class RuleComparatorService {
     if (action === RulePossibility.CONTAINS) {
       try {
         if (!Array.isArray(val2)) {
-          return (val1 as unknown as T[])?.includes(val2);
+          return (val1 as unknown[])?.includes(val2);
```

Comment:
I have a refactor coming soon which will hopefully fix up all of these remaining casts :)